vagrant-spk

```diff
@@ -676,6 +688,28 @@ def getkey(args):
                        # so we make sure to disable TTY allocation.
             )
 
+def vm_subcommand(args):
+    sandstorm_dir = os.path.join(args.work_directory, ".sandstorm")
+    if len(args.command_specific_args) == 0:
+        sys.stderr.write("Try specifying a command to pass through to vagrant, like\n\nvagrant-spk vm up\n\n")
+        sys.exit(1)
+
+    # Do a couple sanity checks.
+    ensure_host_sandstorm_folder_exists()
```

Comment:
No.  `ensure_host_sandstorm_folder_exists()` checks for `~/.sandstorm`, not `<project_folder>/.sandstorm`.

If you think the check below is needed elsewhere, we can extract a `sanity_check_project_sandstorm_folder()` function or something like that.